app/views/users/sessions/new.html.erb

```diff
@@ -1,3 +1,21 @@
 <% title t('landing.page_title') %>
+<div class="govuk-width-container">
+    <h1 class="govuk-heading-xl">
+      <%= service_name %>
+    </h1>
 
-<%= govuk_start_button(text: t('calls_to_action.sign_in'), href: user_azure_ad_omniauth_authorize_path, as_button: true) %>
+    <p class="govuk-body">
+      Use this service to review criminal legal aid applications.
+    </p>
+    <p class="govuk-body">
+      Only caseworkers who have received an email invitation will have access.
+    </p>
+
+    <p class="govuk-body">
+      If you do not have access, you should <%= link_to 'continue using eForms', t('urls.eforms_url') %>.
```

Comment:
No biggie, but just wondering if we want to have `rel: 'external'` in there too.

`<%= link_to 'continue using eForms', t('urls.eforms_url') %>`